test/controllers/area.controller.test.js

```diff
@@ -661,4 +661,111 @@ describe('Area Controller', () => {
       });
     });
   });
+
+  describe('Get all areas and correponding properties', () => {
+    const ajah = AreaFactory.build({ area: 'ajah' });
```

Comment:
Can we add this as a normal case since that is how it will be submitted in the form 🙏?  It looks weird writing the name of an area in lowercase.

```suggestion
    const ajah = AreaFactory.build({ area: 'Ajah' });
```